services/landingpage/compose.yaml

```diff
@@ -0,0 +1,8 @@
+services:
+  landingpage:
+    image: ghcr.io/scicatproject/landingpageserver:latest
+
+    ports:
+      - 4201:80
+
+    user: 0:0
```

Comment:
could we add this block: 

```yaml
profiles: 
 - search
```
as [here](https://github.com/SciCatProject/scicatlive/blob/main/services/searchapi/compose.yaml)